### PR TITLE
feature/add-decision-constants

### PR DESCRIPTION
**Pull request by tohuynh**
**Pull request recommendations:**
- [x] Name your pull request _your-development-type/short-description_. Ex: _feature/read-tiff-files_
- [ ] Link to any relevant issue in the PR description. Ex: _Resolves [gh-12], adds tiff file format support_
- [x] Provide context of changes.
-  Add VoteDecision constants and EventMinutesItemDecision constants.
-  Provide validators for them in the database models
- [x] Provide relevant tests for your feature or bug fix.
- [ ] Provide or update documentation for any feature added by your pull request.

Thanks for contributing!

### CODE REVIEW

**Review by JacksonMaxfield — APPROVED** [review]
:fire: This is so clean. Love it. Merge whenever.

File: cdp_backend/utils/constants_utils.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+    Parameters
+    ----------
+    cls: Type
+        The class to get the class attribute for.
+
+    Returns
+    -------
+    class_attributes: List[Any]:
+        The class attributes values.
```
Comment (JacksonMaxfield) [inline cdp_backend/utils/constants_utils.py:23]: I might recommend changing this and all references to `get_all_class_attr_values`. To me the current name would indicate to me that you are going to return the `attr` _names_ if that makes sense.

**Comment by JacksonMaxfield** [comment]
Oh just to be safe also merge with `main` before merging this branch?

**Comment by tohuynh** [comment]
> Oh just to be safe also merge with `main` before merging this branch?

I think it already is?

**Comment by JacksonMaxfield** [comment]
> > Oh just to be safe also merge with `main` before merging this branch?
> 
> I think it already is? Just Isaac's PR right?

Yep. :shrug: I only see diffs for the files related to this work so sounds good.

**Pull request merged**